### PR TITLE
Update some UI elements

### PR DESCRIPTION
**Pull request by Carbenium**
- Add vertical separator between sidebar and main panel
- Add rounded corners to buttons
- Text box titles are now vertically centered

### CODE REVIEW

**Comment by daniel-beck** [comment]
@Carbenium  Screenshot before/after would be nice for these changes.

**Comment by KostyaSha** [comment]
@daniel-beck could you add this requirement in CONTRIBUTING.md?

**Comment by Carbenium** [comment]
@daniel-beck I'm not a web designer so don't expect too much. ;)

Before | After
---------------------|-----------------------
 | 
 | 
 |

**Comment by KostyaSha** [comment]
imho
:-1: for removing headers background. 
:+1: for centring text
:-1: for removing color of visited links
:-1: for bold, `element name` is just a text, no need make it bold.

**Comment by Carbenium** [comment]
@KostyaSha I really don't see the reason to mark visited links. IMHO this isn't some kind of wiki where you directly want to see where already were.
Regarding the other :-1: : I could live with reverting them.

**Comment by KostyaSha** [comment]
@Carbenium wait for other opinions

**Comment by kzantow** [comment]
@Carbenium some people, navigate to various parts of a web application that they're familiar with by the visited style/color; agree with @KostyaSha : centered text is an improvement; the other changes are not.

**Comment by KostyaSha** [comment]
But if we compare with github, then it doesn't change links color. @Carbenium please don't revert and wait for more feedback

**Comment by tfennelly** [comment]
This is the kind of thing I saw the UI Themes plugin as being useful for. There are just too many different opinions/tastes on this kind of thing, so it's really a wasted effort tweaking this kind of thing because there's always enough people that will not like it. The idea of UI Themes was that each user could have things as they like, or create new themes (like what is effectively happening here) if you don't like any that are there.

**Comment by uhafner** [comment]
I think this PR should be split: 
1) Center the text of labels: this is a bug and should be fixed in core.
2) Background colors: this is a theming thing and should be solved by creating a new theme (this is not possible in the moment biut will be hopefully available with the UI Themes plugin). 
3) Link colors: this is a user configuration (like underlining of a link). This should not be part of a theme, this should be a user property that I can change e.g. in my profile (this is not possible in the moment). (And if this is providided in a theme, it should be configurable by the user.)

**Comment by KostyaSha** [comment]
> 3) Link colors: this is a user configuration (like underlining of a link). This should not be part of a theme, this should be a user property that I can change e.g. in my profile (this is not possible in the moment). (And if this is providided in a theme, it should be configurable by the user.)

-ETOOMANYCONFIGURATIONS

**Comment by KostyaSha** [comment]
> 3) Link colors: 

Do you mean visited links or any links?

**Comment by uhafner** [comment]
Sorry, not for all links, just for the visited link. I updated my comment.
(The color of a link could be part of a theme)

**Comment by KostyaSha** [comment]
I had good experience with IPB, they have 2 parts:
- icons pack
- themes

They provide dropdown chooser in the right bottom, so user can chose theme that he likes. Themes are pre-installed only by admin and allows set default theme, disable theme, add new themes, etc.

**Comment by recena** [comment]
Highlight visited links does make sense in web app. It is different in websites.

File: war/src/main/webapp/css/style.css [hunk 1]
```diff
@@ -74,10 +74,17 @@ body {
   background-color: #f6faf2;
 }
 
+#page-body {
```
Comment (recena) [inline war/src/main/webapp/css/style.css]: :-1: Unnecesary

File: war/src/main/webapp/css/style.css [hunk 2]
```diff
@@ -74,10 +74,17 @@ body {
...
+  min-height: calc(100vh - 103px);
+  overflow: hidden;
+}
+
 #side-panel {
-  padding: 15px 15px 80px 15px;
+  padding: 15px 15px 99999px 15px;
```
Comment (recena) [inline war/src/main/webapp/css/style.css]: Totally :-1:,  Unnecesary

**Comment by recena** [comment]
I totally disagree with the changes proposed in this PR. :-1:

**Comment by oleg-nenashev** [comment]
@Carbenium 
Please don't be offended by lots of :-1:s in the PR. It's just a way to mention that somebody disagrees with the implementation, but it's not a veto.

My recommendation would be to...
1) Submit centering as a separate PR
2) Rework other changes to be a part of a new theme as it has been proposed by @uhafner

**Comment by Carbenium** [comment]
@oleg-nenashev 
That does sound reasonable. I'll see if i can find some time in the next days to split the PR.

**Comment by recena** [comment]
@Carbenium I agree with the @oleg-nenashev's words.

**Comment by recena** [comment]
@Carbenium Any change around the `#page-body`, `#side-panel`, `#main-panel`, `#page-head` and `<footer>` should do carefully. These elements implement the current layout.

**Pull request closed without merging**